casepro/contacts/tests.py

```diff
@@ -122,11 +175,13 @@ def test_as_json(self):
             self.assertEqual(self.ann.as_json(full=False), {'id': self.ann.pk, 'name': "7B7DD8"})
 
     @patch('casepro.test.TestBackend.push_contact')
-    def test_get_or_create_from_urn_no_match(self, mock_push_contact):
+    def test_get_or_create_from_urn(self, mock_push_contact):
```

Comment:
We tend to use one test method per real method because all of our unit tests are integration tests and each new method requires a db transaction to be created and rolled back.